RecoTracker/LSTCore/standalone/code/rooutil/cxxopts.h

```diff
@@ -238,8 +238,7 @@ namespace cxxopts {
   class Option_syntax_exception : public OptionParseException {
   public:
     Option_syntax_exception(const std::string& text)
-        : OptionParseException("Argument " + LQUOTE + text + RQUOTE + " starts with a - but has incorrect syntax") {
-    }
+        : OptionParseException(u8"Argument " + LQUOTE + text + RQUOTE + u8" starts with a - but has incorrect syntax") {}
```

Comment:
@VourMa 
it looks like this broke the standalone in devel

Reply:
Seems like it, even though I am not sure why this change was proposed by the checks. Anyways, trying to fix the checks was a bigger failure that initially thought, sorry about that.

Let me make a standalone PR to fix that.